examples/live/generate_injections.sh

```diff
@@ -0,0 +1,29 @@
+#major work in progress
```

Comment:
I would not split this off to an extra file, I think having it in `run.sh` is fine. Up to you.

I also suggest removing the "major work in progress" comment.

Reply:
This is also prep for the future. I think for doing multiple injections the generate_injections file will wind up getting much more complicated. It may wind up getting switch to a python file instead of a .sh . Just wanted to have this file ready to minimize the amount of changes that need to be done in run.sh for switching to multiple injections.

Reply:
I predict it will get *less* complicated once we use `pycbc_create_injections`, but ok, we can leave this as is.

Reply:
The problem is from what I've seen, pycbc_create_injections does NOT have a way to exactly specify multiple injections. It makes randomized injections from within a specified parameter space. So if we want to continue having the injections exactly match templates in the bank, we will not be able to use pycbc_create_injections